blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -960,6 +960,12 @@ Blockly.ContractEditor.prototype.onClose = function() {
   if (!this.isOpen()) {
     return;
   }
+
+  if (this.disableExamples_ || this.isEditingVariable()) {
```

Comment:
This code behavior change will needed in any case, but could be done (and separately blockly-core unit tested) independently of this set of test changes.